src/lua_console.cpp

```diff
@@ -0,0 +1,86 @@
+#include "lua_console.h"
+
+#include "catalua.h"
+#include "catacharset.h"
+
+lua_console::lua_console() : cWin( newwin( lines, width, 0, 0 ) ), iWin( newwin( 1, width,lines, 0 ) )
+{
+}
+
+void lua_console::dispose()
+{
+    werase( cWin );
```

Comment:
It might be better to put this into the destructor. The destructor is called automatically, and can not be forgotten. (Someone could use `lua_console` without calling `dispose`, leaving the windows allocated.)